src/Cms/Language.php

```diff
@@ -59,6 +60,7 @@ public function __construct(array $props)
 		$this->code         = trim($props['code']);
 		$this->default      = ($props['default'] ?? false) === true;
 		$this->direction    = ($props['direction'] ?? null) === 'rtl' ? 'rtl' : 'ltr';
+		$this->label        = trim($props['label'] ?? Str::upper($this->code));
```

Comment:
This line is not working correctly. The `label` prop from the frontend is always sent as empty string. So if user doesn't enter label, language dropdown shows empty string.

Reply:
Yep, also ran into that. See the possible solutions in the comment above.

Reply:
Oh, I missed that.